test/fixtures/html/basic-include.html

```diff
@@ -6,5 +6,9 @@
   </head>
   <body>
     <!--=include test.html -->
+    <!-- = include test.html -->
```

Comment:
While I see you point, I don't feel it's strong enough to break away from the syntax established by the other tags, especially considering with no change at all you cannot used the required functionality with html includes.